cpp_utils.py

```diff
@@ -346,13 +353,13 @@ def post_struct_member_type_change(member):
     xrefs = filter(lambda x: x.type == ida_xref.dr_I and x.user == 1, xrefs)
     for xref in xrefs:
         if utils.is_func(xref.to):
-            function_ptr_tinfo = idaapi.tinfo_t()
-            ida_struct.get_member_tinfo(function_ptr_tinfo, member)
-            if function_ptr_tinfo.is_funcptr():
+            function_ptr_tinfo = ida_typeinf.tinfo_t()
+            ok = utils.get_member_tinfo(member, function_ptr_tinfo)
```

Comment:
I think that 
```suggestion
            function_ptr_tinfo = utils.get_member_tinfo(member)
```
is better, the function itself would not fail.

Reply:
ack